src/OpenTelemetry.Exporter.ZPages/ZPagesExporter.cs

```diff
@@ -13,10 +13,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 // </copyright>
+
+using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
+using System.Timers;
+using OpenTelemetry.Exporter.ZPages.Implementation;
 using OpenTelemetry.Trace.Export;
+using Timer = System.Timers.Timer;
 
 namespace OpenTelemetry.Exporter.ZPages
```

Comment:
What's the role of exporter? It is not shown on specification: https://docs.google.com/document/d/1vgdbaL7Qb11DFBiF3PzZS9KUJ24VqJuBkKMq3oB-yJQ/edit#

Reply:
I have added it now in the In-Process zPages implementation section